### ISSUE TITLE
range(0,0,1) doesn't iterate

### ISSUE THREAD

**Issue by ingun37** [winterbe/sequency#30]
`range(0,0,1)` doesn't iterate. It should iterate once. It starts working correctly when `endInclusive` is 1. (`range(0,1,1)` iterates twice.)
```typescript
console.log(range(0,0,1).toArray()) // prints []
console.log(range(0,1,1).toArray()) // prints [0, 1]
```

**Comment by couzic** [comment]
Same issue here:
```ts
console.log(range(0, 0).toArray()) // prints [] --- expected [0]
console.log(range(1, 1).toArray()) // prints [] --- expected [1]
console.log(range(2, 2).toArray()) // prints [] --- expected [2]
```
@winterbe do you agree this should be the behavior that most users would naturally expect ? If yes, I can make a pull request.

**Comment by Yona-Appletree** [comment]
I ran into this in the form of `range(1, count) == [ 1, 2 ]` when `count = 2` but yields `[]` when `count = 1` which is clearly wrong so yes, this should absolutely be fixed.

**Comment by winterbe** [comment]
Sorry for the delay guys. It's fixed in version 0.20.0.
